### DISCUSSION TITLE
error An unexpected error occurred: "https://registry.yarnpkg.com/@veramo%2fplugin-identity-manager: Not found".

### DISCUSSION THREAD

**Discussion by HamzaBinSarfraz** [decentralized-identity/veramo#325] [Q&A] [answered]
i am trying to install new veramo dependencies but i am facing an error, i just copy them from veramo website 

`yarn add @veramo/core @veramo/plugin-identity-manager @veramo/plugin-libsodium @veramo/plugin-ethr-did @veramo/plugin-web-did @veramo/plugin-key-manager @veramo/plugin-resolver @veramo/plugin-typeorm @veramo/plugin-w3c`

but @veramo/plugin-identity-manager was not found by yarn, i am using Node-12.19.0

**Comment by gbugyis** [comment, marked as answer]
Hi Muhammad - Thanks for catching that. The node guide needed to be updated. I just went through and made [the changes](https://github.com/uport-project/veramo-website/pull/25). Can you give it another try?  (https://veramo.io/docs/guides/nodejs)

**Reply by vlmold** [reply]
@gbugyis, same issue for react-native page. 
https://veramo.io/docs/guides/react_native

**Reply by HamzaBinSarfraz** [reply]
@gbugyis its working now, thanks

**Reply by gbugyis** [reply]
@vladyslavmunin I'm tackling them one by one, I'll get that one next.

**Reply by HamzaBinSarfraz** [reply]
@gbugyis  I am getting error package `@veramo/plugin-resolver` not found while using [Veramo Create React App](https://veramo.io/docs/guides/browser)
